src/nvim/locale.c

```diff
@@ -3,8 +3,11 @@
 
 // locale.c: functions for language/locale configuration
 
-#include "auto/config.h"
+#include <libintl.h>
```

Comment:
I think this should only be included when `HAVE_WORKING_LIBINTL` is defined.

Reply:
Hmm, in that case we might need a wrapper header for `libintl` to allow IWYU to do this correctly. I'll try it out and we can see if it looks good.

Reply:
Please take a look and determine if the `libintl_wrapper.h` is an OK solution for this.